### ISSUE TITLE
documentation on how to update rules

### ISSUE THREAD

**Issue by client9** [client9/misspell#4]
Currently I just pulled from wikipedia for a word list.

Doco how to add different word lists.

Doco how to manually add words.

**Comment by shurcooL** [comment]
> Doco how to manually add words.

Yes please. How do I? I just ran into an easy mispelling that `mispell` didn't catch.

**Comment by shurcooL** [comment]
Oh, I just realized it didn't catch it for 2 reasons:

1. It's not supported by `misspell`.
2. It was a misspelling in a Go string, not inside a comment. So even after fixing 1, `misspell` still wouldn't find it. :/ Should I still make a PR?

**Comment by client9** [comment]
hi @shurcooL , sorry for the delay.

I'l add doco soon.

You can make misspell look at at everything using

```bash
misspell -source=text 
```

This may mean it tries to correct variable names too.  Sometimes thats good, sometimes bad.


Let me know what you think.

**Comment by shurcooL** [comment]
> You can make misspell look at at everything using `misspell -source=text` Let me know what you think.

That's great, thanks for letting me know.

**Comment by bf4** [comment]
ref https://github.com/client9/misspell/pull/57

**Comment by shawnps** [comment]
+1, would be good to have something in the README or a CONTRIBUTING file on how to update word list so folks can make PRs
